src/schema/schema_create.c

```diff
@@ -739,7 +755,7 @@ __create_table(
     WT_DECL_RET;
     WT_TABLE *table;
     size_t len;
-    int ncolgroups, nkeys;
+    int ncolgroups, num_keys;
```

Comment:
This change is unnecessary.

Reply:
Reverted, thanks! I think I saw that `s_all` was complaining about that variable name, but now I don't see it. I must have confused it with some other error.